api/config.go

```diff
@@ -52,12 +53,14 @@ type Config struct {
 	BaseKey       []byte
 
 	// Swap configs
-	SwapBackendURL string
-	SwapEnabled    bool
+	SwapBackendURL          string
```

Comment:
more explanation in comment needed

Reply:
What do you mean here?

Reply:
please comment exported constants, explain what they are

Reply:
Done